supabase/migrations/20240529104554_remove_driver_overview_downloaded_status.sql

```diff
@@ -0,0 +1,11 @@
+update public.events set new_parcel_status = 'Out for Delivery', event_data = 'with John' where new_parcel_status = 'Driver Overview Downloaded';
+
+delete from public.status_order where event_name = 'Driver Overview Downloaded';
+
+update public.status_order set workflow_order = 12 where event_name = 'Map Generated';
+update public.status_order set workflow_order = 13 where event_name = 'Out for Delivery';

```

Comment:
Looks like `Driver Overview Downloaded` is currently 12 - should we keep it as that and avoid changing other values?

Reply:
I think it would be best to delete the Driver Overview Downloaded status as it now should not be used for anything and has become meaningless

Reply:
Sorry I worded that very badly 😄 I meant set `Out for Delivery` to 12 to replace `Driver Overview Downloaded` and then avoid updating `Map Generated` and all the statuses afterwards so they keep their current order number

Reply:
okay, we could - just so you know we already have a status called "Out for Delivery" with status_order 14, so the result of this is that we'd have a gap in the statuses, ie a status with order 13, a status with order 15, and statuses either side, but none with order 14 - i thought this might be abit confusing to future devs, although it won't affect the code

![image](https://github.com/NorwoodAndBrixtonFoodbank/nbf-website/assets/138508884/f1f6b637-0482-4358-9bb6-830d6b45cd47)

Reply:
Yeah just noticed - did we update the status to old "Out for Delivery"? I agree with you on the gap but I'm confused by 2 out for deliveries even more

Reply:
Is this just something that exists in seeded data but we can't manually update to it and there's no event that triggered it (until now where downloading the driver overview changes it)?

Reply:
No there is only 1 "Out for Delivery" now
![image](https://github.com/NorwoodAndBrixtonFoodbank/nbf-website/assets/138508884/8982f474-53f8-4122-933c-3039e9106297)

Reply:
Ok so to summarize
- We had Driver Overview Downloaded and Out for Delivery
- Status updated to Driver Overview Downloaded when you download the pdf
- Out for Delivery doesn't seem to be used anywhere and just appears in the seeded data?
- You used the unused Out for Delivery to update to when we download the pdf and deleted Driver Overview Downloaded
- You shifted the numbers so there's no gap where Driver Overview Downloaded was
Is this correct?
If yes then happy with stuff, guess just worried about Out for Delivery being used for something else

Reply:
Yes that's correct - Out for Delivery was not used elsewhere